log4j-1.2-api/src/main/java/org/apache/log4j/builders/filter/LevelRangeFilterBuilder.java

```diff
@@ -83,19 +86,23 @@ public Filter parse(PropertiesConfiguration config) {
     }
 
     private Filter createFilter(String levelMax, String levelMin, boolean acceptOnMatch) {
-        Level max = Level.FATAL;
-        Level min = Level.TRACE;
+        Level max = Level.OFF;
+        Level min = Level.ALL;
         if (levelMax != null) {
-            max = OptionConverter.toLevel(levelMax, org.apache.log4j.Level.FATAL).getVersion2Level();
+            max = OptionConverter.toLevel(levelMax, org.apache.log4j.Level.OFF).getVersion2Level();
         }
         if (levelMin != null) {
-            min = OptionConverter.toLevel(levelMin, org.apache.log4j.Level.DEBUG).getVersion2Level();
+            min = OptionConverter.toLevel(levelMin, org.apache.log4j.Level.ALL).getVersion2Level();
         }
         org.apache.logging.log4j.core.Filter.Result onMatch = acceptOnMatch
                 ? org.apache.logging.log4j.core.Filter.Result.ACCEPT
                 : org.apache.logging.log4j.core.Filter.Result.NEUTRAL;
 
```

Comment:
Is it better to output warning log if `max <  min` here?
Current released implementation should work if we swap `levelMin` and `levelMax` in `log4j.properties`.
This is degradation for people who configure log4j like that.

Reply:
There was no warning in Log4j 1.x, so I wouldn't add it here. You can add a warning in the Log4j2 factory `LevelRangeFilter#createFilter` (using the `StatusLogger`).

Reply:
OK, I won't add warning log in Log4j2 factory too because if someone misconfigures LogLevelRangeFilter, no logs will be output - they should be able to notice misconfiguration easily.